website/content/api-docs/secret/databases/cassandra.mdx

```diff
@@ -42,23 +42,35 @@ has a number of parameters to further configure a connection.
 - `insecure_tls` `(bool: false)` – Specifies whether to skip verification of the
   server certificate when using TLS.
 
-- `tls_server_name` `(string: "")` – Specifies the name to use as the SNI host when 
+- `tls_server_name` `(string: "")` – Specifies the name to use as the SNI host when
   connecting to the Cassandra server via TLS.
-  
+
 - `pem_bundle` `(string: "")` – Specifies concatenated PEM blocks containing a
   certificate and private key; a certificate, private key, and issuing CA
-  certificate; or just a CA certificate.
-
-!> **Known Issue:** There is a known issue when using `pem_bundle` with only a CA (no client certificate & key)
-   where Vault will not parse the CA certificate correctly. To work around this, use `pem_json` with the
-   following structure: `{"ca_chain": ["-----BEGIN CERTIFICATE-----\nMIIEFjC...FNYakP7I\n-----END CERTIFICATE-----"]}`
-   Also make sure the PEM data is properly JSON encoded with `\n` instead of newlines.
+  certificate; or just a CA certificate. Only one of `pem_bundle` or `pem_json` can be specified.
```

Comment:
Do you think there's value in providing a sample command for constructing this concatenated bundle to minimize the chances of a malformed bundle being provided?

Reply:
I lean towards no because such a command is going to be fairly specific to my development environment.